apis/Google.Cloud.Dialogflow.V2/Google.Cloud.Dialogflow.V2/Agent.cs

```diff
@@ -1197,7 +1216,7 @@ public sealed partial class SearchAgentsRequest : pb::IMessage<SearchAgentsReque
     public const int PageTokenFieldNumber = 3;
     private string pageToken_ = "";
     /// <summary>
-    /// Optional. The next_page_token value returned from a previous list request.
+    /// The next_page_token value returned from a previous list request.
```

Comment:
@jskeet Flagging this as well as a potential issue to follow up on. Here, the `Optional` was removed from comments but the `Required` was not added. You can see that in similar scenarios in this same file (see line 1698 for instance), `Optional` has been changed by `Required`. Not a big deal though.

Reply:
Interesting. My guess is that it's because there isn't a `google.api.field_behavior` annotation on the proto at all. There probably should be. I'll add a comment on the CL.